### ISSUE TITLE
Non-latin file names are truncated

### ISSUE THREAD

**Issue by sikmir** [afify/sfm#41]
Narrow window:
![scr1](https://user-images.githubusercontent.com/688044/151914527-fdd168e4-8b49-4dc7-ace2-e9734a6cacbb.jpg)
Wider window:
![scr2](https://user-images.githubusercontent.com/688044/151914529-1114db87-21cd-4c0a-b1b2-dd578dedc533.jpg)

**Comment by afify** [comment]
@sikmir please check beb91ae

**Comment by sikmir** [comment]
Thanks! It's OK now.

**Comment by sikmir** [comment]
I wonder, why this commit is not in the main branch? GitHub says "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

**Comment by sikmir** [comment]
@afify Is there any plans for v0.5 release? It would be really great to get this bug fixed.

**Comment by afify** [comment]
> @afify Is there any plans for v0.5 release? It would be really great to get this bug fixed.

@sikmir Yeah 0.5 is under development with rewrite most of the code for more performance improvements and bug fixes.
